Assets/Scripts/Event Manager.cs

```diff
@@ -0,0 +1,53 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Events;
+
+
+[CreateAssetMenu(fileName = "EventManager", menuName = "Scriptable Objects/EventManager")]
+public class EventManager : ScriptableObject
+
+{
+    public static EventManager instance;
+
+    public delegate void ButtonClickEvents();
+    public ButtonClickEvents onForwardClick;
+    public ButtonClickEvents onBackwardClick;
+
+
+    public delegate void SceneEvent();
```

Comment:
Its really good that you separate out this delegate as its own type from ButtonClickEvents.